site/_posts/2022-11-01-authenticators-ceremonies-webauthn-oh-my.md

```diff
@@ -0,0 +1,128 @@
+
+### The Authentication Flow
+
+After a user's device is registered with the RP, the user can be authenticated. There are six steps involved in the WebAuthn authentication process:
+
+{% include _image.liquid src="/assets/img/blogs/webauthn-terminology/authentication-flow.png" alt="WebAuth Authentication Flow diagram" class="img-fluid" figure=true %}
+
+The following are the steps for this authentication process:
+
+1. The user initiates login via the browser.
+2. The server (RP) creates and sends a unique challenge to the authenticator.
+3. The authenticator receives the challenge.
+4. The authenticator prompts the user for device authorization (again, via biometrics, facial recognition system, etc.).
+5. Upon successful authorization, the authenticator creates a cryptographic signature and sends it back to the server.
+6. The server verifies the signature of the unique challenge that it sent to the user (in step 2). Upon successful verification, the user is logged into the web application.
+
+## WebAuthn vs. FIDO 2
+
+WebAuthn was developed by the [World Wide Web Consortium (W3C)](http://www.w3.org/), which is the international standards organization for the world wide web. Meanwhile, Fast Identity Online 2 (FIDO2) was created by [FIDO Alliance](https://fidoalliance.org/) with the aim of creating passwordless authentication. WebAuthn is a core component of [FIDO2 specifications](https://fidoalliance.org/fido2/).
+
+The FIDO Alliance decided to partner with W3C to make FIDO2 authentication a standard for the web. WebAuthn became a part of the FIDO2 specification, and a [set of browser APIs](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API) was finalized, which is widely used today for passwordless authentication.
+
+## Common WebAuthn Terminology
+
+If you're looking to enter into the WebAuthn space, it can be helpful to understand the following terminology:
+
+### Cross-Platform vs. Single-Platform
+
+[Cross-platform authenticators](https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Platform_vs_Cross-Platform.html.) (also known as roaming authenticators) are authenticators that can be used with another device, like a [YubiKey](https://www.yubico.com/), to perform authentication.
+
+[Single-platform authenticators](https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Platform_vs_Cross-Platform.html) (also known as platform authenticators) are authenticators that are built into the device, like Android pattern gestures or biometrics, iOS Face ID, or voice unlocks.
+
+### User Presence vs. User Verification 
+
+User presence indicates the user's current state (*ie* whether someone is present). For instance, an authenticator may have a button a person pushes to indicate they are in possession of it. In this situation, a user is present, but as a relying party, you don't know which user. 
```

Comment:
It may be worth mentioning that a test of user presence is required for all WebAuthn ceremonies.